gensrc/thrift/Types.thrift

```diff
@@ -66,7 +66,7 @@ enum TPrimitiveType {
   DATE,
   DATETIME,
   BINARY,
-  DECIMAL,
+  DECIMAL_DEPRACTED, // not used now, only for place holder
```

Comment:
Is it compatible to change the enum's name?

Reply:
we don't use name literal in our code , just use as int